content/onionservices/onion-service-asking-for-private-key/contents.lr

```diff
@@ -0,0 +1,9 @@
+_model: question
+---
+title: Why is X.onion asking for a private key?
+---
+seo_slug: onion-service-asking-for-private-key
+---
+description:
+If you see this, then you're trying to access an authenticated onion. An authenticated onion service is an onion service that requires you to provide an authentication token (in this case, a private key) before accessing the service.
+[Learn more about onion authentication.](https://tb-manual.torproject.org/onion-services/#client-auth) 
```

Comment:
The way this paragraph reads, people might be confused that the private key is provided (i.e. transmitted) to the service. Perhaps it's worth clarifying in a sentence that "The private key is not transmitted to the service, and it's only used to decrypt its descriptor locally", or a variant of this.

Reply:
Fixed!